README.md

```diff
@@ -1 +1,10 @@
-# e-banking
\ No newline at end of file
+# App Digital Bank 
```

Comment:
This description should assume the user does not necessarily have the tools necessary for running the tests. Consider adding a pre-requisites section that mentions the need for having node.js installed in their system.

Also, they need to install the dependencies stated in your package.json before being able to run the tests: (That's achieved by running: `npm install` on your project's root directory). Only after that they'll be able to run the tests.